lib/list.js

```diff
@@ -31,6 +31,12 @@ function List (key, options) {
 		defaultColumns: '__name__',
 	};
 
+  // Disable the autoIndex option in production
+  // https://mongoosejs.com/docs/guide.html#autoIndex
+  if (keystone.get('env') === 'production') {
+    defaultOptions.schema = Object.assign(defaultOptions.schema, { autoIndex: false })
+  }
+
```

Comment:
Is it intended to turn off the `autoIndex` of the schema of `List` only? Or can we turn off the `autoIndex` globally on connecting DB since it's not for production?

https://github.com/twreporter/keystone/blob/master/lib/core/openDatabaseConnection.js
https://mongoosejs.com/docs/guide.html#indexes

Reply:
Both solutions are OK for me. I put this in the `List` function as it seemed to me that this was the only function to create/update model. Did I take it the wrong way?

Reply:
I had not checked if there's any other function will create index for models of in mongo, so for me it's more straightforward to close all the `autoIndex` when connecting to DB in production. But if `List` is the only or the main function which touching the model, setting `autoIndex` in `List` as `false` will work well also as you said.

Reply:
If `autoIndex` is not recommended in the mongoose docs, we could close all the `autoIndex` when we connecting to DB in production.

Reply:
I will configure the option during connection if that makes more sense to us.